src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -487,6 +487,11 @@ private function registerFragmentsConfiguration(array $config, ContainerBuilder
 
             return;
         }
+        if ($container->hasParameter('fragment.renderer.hinclude.global_template') && (null !== $container->getParameter('fragment.renderer.hinclude.global_template') && null !== $config['hinclude_default_template'])) {
+            throw new \LogicException('Ambiguous configuration detected, prior to Symfony 4.3 you should not use both config keys ("templating.hinclude_default_template" and "fragments.hinclude_default_template", please use "fragments.hinclude_default_template".');
```

Comment:
`You cannot set both "templating.hinclude_default_template" and "fragments.hinclude_default_template", please only use "fragments.hinclude_default_template".`